### PR TITLE
Development environment setup

### PR DESCRIPTION
**Pull request by mershark**
In this task, I accomplished the following tasks in alignment with the project requisites:
- Created a new repository.
- Adhered to the Gitflow workflow.
- Generated a fresh React application.
- Integrated linters for code quality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,171 @@
...
+## 🔭 Future Features <a name="future-features"></a>
+- [ ] **Calculator Papge**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (roshan-bajgain) [inline README.md]: - [ ] When I try to open `Issue page` then the link is broken please use the correct URL here `https://github.com/mershark/math-magician/issues` as it makes you README file more professional.

File: README.md [hunk 2]
```diff
@@ -1 +1,171 @@
...
+
+([coming soon]())
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo Website <a name="live-demo"></a>
+
+([coming soon]())
+
```
Comment (roshan-bajgain) [inline README.md]: - [ ] As this makes your README file not more professional So I would suggest please remove this `video and live Demo `link as you did not have a link right now. In the future, if you decide to put the link you can put it with a video and live demo link. So for now please remove it. It make more descriptive README file.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @mershark, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- Gitflow is followed. ✔️
- PR has a good title and summary. ✔️
- App is running without any error ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — APPROVED** [review]
# Hi @mershark, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Webapp is properly setup with CRA ✔
- Webapp runs as expected ✔
- Correct use of Git Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**